packages/dids/package.json

```
@@ -77,7 +77,7 @@
     "@decentralized-identity/ion-pow-sdk": "1.0.17",
     "@decentralized-identity/ion-sdk": "1.0.1",
     "@web5/common": "0.2.1",
-    "@web5/crypto": "0.2.2",
+    "@web5/crypto": "0.2.3",
```

`@web5/dids` needs to be updated before using `@web5/crypto` version `0.2.3` so reverting.
Details here: https://github.com/TBD54566975/web5-js/pull/318
```suggestion
    "@web5/crypto": "0.2.2",
```